frontend/src/app/features/work-packages/components/wp-activity/comment-service.ts

```diff
@@ -29,17 +29,20 @@
 import { Injectable } from '@angular/core';
 import { I18nService } from 'core-app/core/i18n/i18n.service';
 import { ToastService } from 'core-app/shared/components/toaster/toast.service';
-import { Subject } from 'rxjs';
+import {
+  BehaviorSubject,
+  Subject,
+} from 'rxjs';
 import { WorkPackageNotificationService } from 'core-app/features/work-packages/services/notifications/work-package-notification.service';
 import { WorkPackageResource } from 'core-app/features/hal/resources/work-package-resource';
 import { HalResource } from 'core-app/features/hal/resources/hal-resource';
 
 @Injectable()
 export class CommentService {
-  // Replacement for ng1 $scope.$emit on activty-entry to mark comments to be quoted.
-  // Should be generalized if needed for more than that.
   public quoteEvents = new Subject<string>();
 
+  public draft = new BehaviorSubject<string|null>(null);
```

Comment:
both this and the `quoteEvents ` subject above should be dollar-prefixed (or postfixed?), though I'm unsure if the effort is it